src/installation/guides/fde.md

```diff
@@ -122,27 +122,9 @@ On a UEFI system, the EFI system partition also needs to be mounted.
 
 Before we enter the chroot to finish up configuration, we do the actual install.
 
-`xbps-install` might ask you to [verify the RSA
-keys](../../xbps/troubleshooting/common-issues.md#verifying-rsa-keys) for the
-packages you are installing.
-
-```
-# xbps-install -Sy -R https://alpha.de.repo.voidlinux.org/current -r /mnt base-system lvm2 cryptsetup grub
-[*] Updating `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
-x86_64-repodata: 1661KB [avg rate: 2257KB/s]
-`https://alpha.de.repo.voidlinux.org/current' repository has been RSA signed by "Void Linux"
-Fingerprint: 60:ae:0c:d6:f0:95:17:80:bc:93:46:7a:89:af:a3:2d
-Do you want to import this public key? [Y/n] y
-130 packages will be downloaded:
-...
-```
-
-UEFI systems will have a slightly different package selection. The installation
-command for a UEFI system will be as follows.
-
-```
-# xbps-install -Sy -R https://alpha.de.repo.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi lvm2
-```
+Follow the instructions on [Base Installation](chroot.md#base-installation) to
+install the `base-system` and, additionally, the other important packages we
+need in this specific case, those are: `lvm2` and `cryptsetup`.
```

Comment:
```suggestion
Follow the instructions for a [base installation](chroot.md#base-installation) to
install the `base-system` package together with the packages needed for this guide: `lvm2` and `cryptsetup`.
```

Reply:
This is missing the mention of GRUB UEFI.

Reply:
It's because, as I said before, to be more consistent with `chroot.md`, grub could be mentioned only later on as one of the last steps of the installation, and this would help by removing the mentions to modifications on ´/etc/default/grub´ on separate steps of the installation as well, by putting then all together at the end.
This could probably help further improvements of repetition removal on the `fde.md` guide in the future too, since the more consistent the flows of installation are with each other the easier it is to spot duplicated parts.